src/modules/social/posts/posts.schema.ts

```diff
@@ -1,11 +1,16 @@
 import { z } from "zod"
-import { buildJsonSchemas } from "fastify-zod"
 
 export const postCore = {
-  title: z.string(),
-  body: z.string(),
-  tags: z.string().array().optional(),
-  media: z.string().url().optional()
+  title: z.string({
+    invalid_type_error: "Title must be a string",
+    required_error: "Title is required"
+  }),
+  body: z.string({
+    invalid_type_error: "Body must be a string",
+    required_error: "Body is required"
```

Comment:
I think it says in the docs that this is optional

Reply:
You may be right. Need to double-check where the schema is used. The body is required when creating a post, but may not be when updating one.

Reply:
Should be resolved by 12cbed02176da0b88547b5c13e3ca48dfc6c4e58